src/model_filereading.cpp

```diff
@@ -106,10 +105,14 @@ void Model::readFileXYZ(const std::string& filepath){
     if (isAtomLine(substrings)) {
 
       std::string valid_symbol = strToValidSymbol(substrings[0]);
+      if (valid_symbol.empty()) {
+        invalid_symbol_detected = true;
+        continue;
+      }
       atom_amounts[valid_symbol]++; // adds one to counter for this symbol
 
       // as a safety mechanism, if an element symbol is assigned two atomic numbers, default to 0
-      // so it becomes apparent laster on that something is wrong
+      // so it becomes apparent later on that something is wrong

```

Comment:
I think `elem_Z` is not really used at the moment so I don't know what this safety mechanism will change. There is a similar safety mechanism in `readFilePDB()`. Better to keep elem_Z for future uses though.